app/views/admin/partners/_form.html.erb

```diff
@@ -15,7 +15,9 @@
       <% end  %>
     <% end  %>
     <h2>Basic Information</h2>
-    <%= f.input :name, class: "form-control" %>
+    <%= f.input :name, class: "form-control", input_html: { 'data-controller': "partner-form-validation", 'data-partner-form-validation-target': 'source' } %>
+    <p class="invalid-feedback" id="partner-name-feedback">This partner name has already been taken.</p>
```

Comment:
I'm getting a double warning 

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/4e9299de-a8ca-4ad9-acd6-6c15fa8beb82)

Reply:
Yeah i think this would be a tricky fix as the JS would have to figure out if the warning was already given from the failed request as plain old HTML.

Reply:
could it link to the partner with the same name or is that a huge faff with other implications around who can see the clashing partner? would be one way to make the warnings a bit diff at least. but i think this is fine as is too.

Reply:
That is a neat idea but I think it may be best written in a new issue with clear specs